packages/zoe/src/zoe.js

```diff
@@ -318,8 +522,21 @@ const makeZoe = (additionalEndowments = {}) => {
         },
       ]),
     );
-    handleToSeat.init(inviteHandle, seat);
-    const invitePayment = inviteMint.mintPayment(inviteAmount);
+    handleToSeatFn.init(inviteHandle, seatFn);
+    return inviteMint.mintPayment(inviteAmount);
+  };
+
+  // TODO(msm): remove this deprecated one in this PR
+  // TODO don't merge until this one is gone or mind changes
+  const makeInvitePair = (
```

Comment:
I agree that it should happen very soon. But does it need to happen in this PR?

(Also, after talking to @warner and @Chris-Hibbert I'm pursuing a hypothesis that our integration test failure is because it uses the dapp-simple-exchange which is a separate repository compat with current master. To ease the version transition, I'm going to see if emulating exactly the old API makes things work again. If so, the intermediate compat-with-both nature of this PR may help us deal with the version slippage.)

Reply:
Now that we know we need to support the old API in this PR, we can also wait to upgrade the opera ticket contract till the next PR.